dvc/cache/base.py

```diff
@@ -324,15 +324,22 @@ def create_tasks(executor, amount):
                     yield rel_path_infos.pop(task), task.result()
 
     def _transfer_directory(
-        self, from_tree, from_info, jobs, no_progress_bar=False
+        self, from_tree, from_info, out_info, jobs, no_progress_bar=False
     ):
-        dir_info = DirInfo()
+        if out_info:
+            initials = out_info.parts[-1:]
+        else:
+            initials = ()
```

Comment:
Could you elaborate on what's going on here and why we need the initials, please?

Reply:
this logic is gone in my latest revision, apparently, I didn't even commit it but should be now up to date